src/core/utils/resultsutils.js

```diff
@@ -0,0 +1,17 @@
+import SearchStates from '../storage/searchstates';
+
+/**
+ * Returns a CSS class for the input searchState
+ * @param {string} searchState - A search state from {@link SearchStates}
+ * @returns {string}
+ */
+export function getContainerClass (searchState) {
+  switch (searchState) {
+    case SearchStates.PRE_SEARCH:
+      return 'yxt-Results--preSearch';
+    case SearchStates.SEARCH_LOADING:
+      return 'yxt-Results--searchLoading';
+    case SearchStates.SEARCH_COMPLETE:
+      return 'yxt-Results--searchComplete';
```

Comment:
looks like this can return `undefined`, can we get a default case?

Reply:
Should we throw an error here or add an undefined searchState?